practico_01/ejercicio_10.py

```diff
@@ -23,7 +26,12 @@ def tiene_pares_for_else(numeros: Iterable[int]) -> bool:
     """Re-Escribir utilizando for-else con dos return y un break.
     Referencia: https://docs.python.org/3/tutorial/controlflow.html#break-and-continue-statements-and-else-clauses-on-loops
     """
-    pass # Completar
+    for num in numeros:
+        if num % 2 == 0:
+            return True
+            break
```

Comment:
Incorrecto el return True debería estar fuera del For. El break no se ejecuta nunca así